test/README.testsuite

```diff
@@ -305,3 +307,28 @@ in the test definition (i.e. TEST-*-NAME/test.sh), which will skip the post-test
 check for missing coverage for the respective test.
 
 [0] https://coveralls.io/github/systemd/systemd
+
+Running fuzzers locally
+=======================
+
+When introducing a new fuzzer you usually want to check how well it behaves, as
+well as to check if it triggers anything unusual straight away. The easiest
+way is to use the LLVM's libFuzzer engine [0] together with ASan and UBSan:
+
+$ CC=clang CXX=clang++ meson setup build-libfuzz -Dllvm-fuzz=true -Db_sanitize=address,undefined -Db_lundef=false
+$ ninja -C build-libfuzz
```

Comment:
`ninja -C build-libfuzz fuzzers` should speed it up a bit.

I think
```sh
SANITIZER=address,undefined ./tools/oss-fuzz.sh
```
was supposed to be used to build the fuzz targets locally. It also sets `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION` as far as I can remember.

Reply:
I'd also mention `UBSAN_OPTIONS=print_stacktrace=1:print_summary=1:halt_on_error=1` somewhere. Without that the fuzz targets don't fail under UBSan.

Reply:
I knew I saw the `oss-fuzz.sh` stuff somewhere, and it's, of course, already documented in `docs/HACKING.md`, so I updated it with the stuff from this commit (and hopefully addressed the comments as well).